specs/playerPageTCs.js

```diff
@@ -0,0 +1,26 @@
+describe('Player Page Tests', () => {
+
+    // let searchField = $('[placeholder="Enter search keywords"]')
+    let playerFullName = global.importedPlayerTag.playerFullName;
+    let playerCountry = global.importedPlayerTag.playerCountryName;
+    
+    it('Displayed Player Name matches feed', () => {
+        
+        browser.get('players/player.08793.tiger-woods.html');
```

Comment:
This action is common for both test cases, so it will be better to move this line to `beforeEach` block - just after `describe`:
```
beforeEach(() => {
    // Here will be preconditions, they are same for all tests in this spec / suite
    browser.get('players/player.08793.tiger-woods.html');
});
```